app/src/main/java/com/example/chicoryaos/ui/bookmark/BookmarkViewHolder.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.chicoryaos.ui.bookmark
+
+import androidx.recyclerview.widget.RecyclerView
+import com.example.chicoryaos.databinding.ItemBookmarkProductBinding
+
+class BookmarkViewHolder(private val binding: ItemBookmarkProductBinding) :
+    RecyclerView.ViewHolder(binding.root) {
+
+    fun onBind(bookmarkData: Bookmark) {
+        binding.ivItemWishList.setImageResource(bookmarkData.productImage)
+        binding.tvItemWishListTitle.text = bookmarkData.productTitle
+        binding.tvItemSalePrice.text = bookmarkData.productPrice.toString()
+    }
```

Comment:
중요한 부분은 아니지만 scope함수 with(binding) { } 으로
한번에 묶어서 사용할 수도 있습니다!

```suggestion
    fun onBind(bookmarkData: Bookmark) {
    with(binding) {
 ivItemWishList.setImageResource(bookmarkData.productImage)
        tvItemWishListTitle.text = bookmarkData.productTitle
        tvItemSalePrice.text = bookmarkData.productPrice.toString()
        }
    }
```

Reply:
오호 좋은 팁 감사합니다!!